.travis.yml

```diff
@@ -0,0 +1 @@
+language: java
```

Comment:
why do we have a .travis file?

Reply:
Forgot to remove it, I was just playing with Travis.